goldens/golden_test.go

```diff
@@ -119,3 +124,21 @@ func showTopLevel(dir string) (string, error) {
 	b, err := exec.Command("git", "-C", dir, "rev-parse", "--show-toplevel").Output()
 	return strings.TrimSpace(string(b)), err
 }
+
+func discoverWantStderr(want string) []string {
```

Comment:
Super nit: "discover" feels a bit atypical of a verb to use here. Certainly my first thought was "what does that mean?" This is extracting desired STDERR lines from the golden file. Maybe we should just be a little more verbose, like calling the function `extractWantStderrFromGolden`?

Or for that matter, I kinda wonder if the goldens could just be split into two files so that stderr isn't in the same file. It is a bit confusing to have STDERR in the `.out` files considering it is not part of the output file.